python/lsst/obs/lsst/cli/cmd/commands.py

```diff
@@ -35,7 +35,7 @@
 from ... import script
 
 
-defaultRegex = r".*Photodiode_Readings.*txt\b"
+defaultRegex = r".*Photodiode_Readings.*txt|.*photodiode.ecsv\b"
```

Comment:
I wonder if `$` could be used instead of `\b`? Something like

```suggestion
defaultRegex = r"(Photodiode_Readings_[0-9_]+\.txt|photodiode\.ecsv)$"
```
? The leading `.*` aren't really needed).